js/everything.js

```diff
@@ -630,6 +641,23 @@ $(function() {
     });
     // Load the player
     ZenPlayer.init(currentVideoID);
+
+    $(document).on("keyup", function(evt) {
+        evt.preventDefault();
```

Comment:
Se my comment for the `keydown` event handler. We need to make sure we don't play/pause while the user is typing in the search box